### ISSUE TITLE
Can't ignore folders

### ISSUE THREAD

**Issue by alessio-b-zak** [sp1thas/dropboxignore#22]
Using dropboxignore I am able to ignore files but not folders.

On creating a new folder inside a dropbox synced folder and ignoring
```
mkdir test
dropboxignore ignore ./test
```
The command outputs
```
Total number of ignored files: 0 
Total number of ignored folders: 0
```
and the folder is not ignored and remains synced to the online repository.

using `-v 2` the command outputs 
```
Thu Sep  9 16:46:40 BST 2021  [ DEBUG ] Operating system: Linux 
Thu Sep  9 16:46:40 BST 2021  [ DEBUG ] realpath command is installed 
Thu Sep  9 16:46:40 BST 2021  [ DEBUG ] attr package is installed 
Thu Sep  9 16:46:40 BST 2021  [ DEBUG ] Input folder: "./test" 

Total number of ignored files: 0 
Total number of ignored folders: 0 
```

I am using Ubuntu 18.04, dropboxignore v1.0.0,  dropbox CLI 2020,03.04 and dropbox daeman 130.4.4978. 

Any help is greatly appeciated

**Comment by sp1thas** [comment]
Hello @alessio-b-zak 

First of all thank you for your feedback!

I can verify that this is a bug, it seems that dropboxignore is not working properly in this case.

I just released a patch `v1.0.1`. Feel free to reinstall from master or update via snap: `$ snap refresh dropboxignore`

**Comment by alessio-b-zak** [comment]
Thanks, can confirm that everything is working :)
